lib/structures/Message.js

```diff
@@ -120,21 +120,22 @@ class Message extends Base {
             this.interaction = null;
         }
 
-        if(this.channel.guild) {
+        if(data.guild_id) {
+            const guild = this._client.guilds.get(data.guild_id);
```

Comment:
This will cause uncached guilds to throw errors

Reply:
Oh wopsie, added a check for uncache guild